src/internal/model_render.go

```
@@ -253,71 +253,101 @@
+	}
+	for _, line := range lines {
+		r.AddLines(line)
+	}
+	return r.Render()
+}
+
+
+func (m *model) ensureMetadataLoaded() {
+	if len(m.fileMetaData.metaData) == 0 &&
+		len(m.fileModel.filePanels[m.filePanelFocusIndex].element) > 0 &&
+		!m.fileModel.renaming {
```

golangci-lint (1.64.8)

unnecessary leading newline

(whitespace)